### ISSUE TITLE
Accept any url for upstream

### ISSUE THREAD

**Issue by splix** [ETCDEVTeam/emerald-cli#57]
Currently it requires an IP and a port, which works fine for a local node, but doesn't work for a remote one that uses https for example. Let's use just an URL instead of IP:PORT. Like:
```
emerald server --upstream https://web3.gastracker.io/morden
```

**Comment by r8d8** [comment]
@splix `server` command doesn't accepts `--upstream` argument. Do you mean `transaction` command?

**Comment by splix** [comment]
sorry, messed everything. I mean allow `--upstream` argument for server mode too, instead of host:port

**Comment by r8d8** [comment]
Implemented in #94
